uint8 status

uint8 REJECTED = 0
uint8 ACCEPTED = 1
uint8 REWORK   = 2